src/Compiler/SyntaxTree/SyntaxTrivia.fs

```diff
@@ -115,6 +115,14 @@ type SynExprMatchBangTrivia =
 [<NoEquality; NoComparison>]
 type SynExprAnonRecdTrivia = { OpeningBraceRange: range }
 
+[<NoEquality; NoComparison>]
+type SynExprSequentialTrivia =
+    {
+        SemiColonRange: range option
+    }
+
+    static member Zero = { SemiColonRange = None }
```

Comment:
I see that the other types defined in this file also use `static member`. Would it make sense to use `static member val` so that new record instances are not allocated each time `Zero` is accessed? (I don't know how or how often this `Zero` member is used.)

Reply:
> I don't know how or how often this Zero member is used

I don't think that much, but yeah fair take.